src/index.ts

```diff
@@ -19,29 +19,20 @@
 
 import 'reflect-metadata'
 
+export * from './asserts/AssertCollection'
 export * from './asserts/AssertNotNull'
 
 export * from './domain/behaviours/BacktrackingBehaviour/BacktrackingBehaviour'
 export * from './domain/behaviours/BacktrackingBehaviour/BasicBacktrackingBehaviour'
 export * from './domain/behaviours/BacktrackingBehaviour/HierarchicalIterStack'
-
 export * from './domain/behaviours/IBehaviour'
 
-export * from './domain/DateFormat'
-
 export * from './domain/exceptions/InvalidChoiceException'
 export * from './domain/exceptions/NotImplementedException'
 export * from './domain/exceptions/PromptValidationException'
 export * from './domain/exceptions/ResourceNotFoundException'
 export * from './domain/exceptions/ValidationException'
 
-export * from './domain/SupportedContentType'
-export * from './domain/FlowRunner'
-export * from './domain/IdGeneratorUuidV4'
-export * from './domain/IFlowRunner'
-export * from './domain/IIdGenerator'
-export * from './domain/IResourceResolver'
-
 export * from './domain/prompt/BasePrompt'
```

Comment:
@bulatgab , if we have to follow a specific order here to avoid an error, I'd suggest we leave a note in a comment.

- [ ] leave comment

but for curiosity, could you tell us the error ?

Reply:
@safydy-r About the actual error I had - it was just `IReadBlock` missing from exports.

Everything else was additional - I rearranged the exports so they would follow the order in the file system (and be alphabetically sorted), this way it was easier to see if anything was missing (and some other things were indeed).